llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -407,6 +407,53 @@ def SYNTACORE_SCR7 : RISCVProcessorModel<"syntacore-scr7",
                                                FeatureStdExtZkn],
                                               [TuneNoDefaultUnroll, FeaturePostRAScheduler]>;
 
+def TENSTORRENT_ASCALON_D8 : RISCVProcessorModel<"tt-ascalon-d8",
+                                                 NoSchedModel,
+                                                 [Feature64Bit,
```

Comment:
You can include all the RVA23U64 features with a !listconcat, similar to what the other processors have done with RVA22U64

Reply:
Thanks, addressed